cmd/spire-server/cli/bundle/delete.go

```diff
@@ -34,6 +44,7 @@ func (c *deleteCommand) synopsis() string {
 
 func (c *deleteCommand) appendFlags(fs *flag.FlagSet) {
 	fs.StringVar(&c.id, "id", "", "SPIFFE ID of the trust domain")
+	fs.StringVar(&c.mode, "mode", deleteBundleRestrict, "Deletion mode: one of restrict, delete, or dissociate")
```

Comment:
Maybe it make sense to use the constants also to build the usage string?

Reply:
sure, why not! commit coming up.